webapp/sources/ldap-inventory/inventory-api/pom.xml

```diff
@@ -55,5 +55,9 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
       <groupId>org.bouncycastle</groupId>
       <artifactId>bcpkix-${bouncycastle-compat}</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.comcast</groupId>
+      <artifactId>ip4s-core_${scala-binary-version}</artifactId>
+    </dependency>
```

Comment:
it was needed to move the `INetAddress` codec from `NodeFacts` (from rudder-core)